src/main/java/classify/user/InputParsing.java

```diff
@@ -1,13 +1,7 @@
 package classify.user;
 
-import classify.commands.AddStudent;
-import classify.commands.Commands;
-import classify.commands.FileIOCommands;
-import classify.commands.DeleteCommands;
-import classify.commands.ListStudentsCommand;
-import classify.commands.StudentSorter;
-import classify.commands.ViewStudent;
-import classify.commands.EditStudent;
+import classify.commands.*;
```

Comment:
perchance import the specific class instead of using *?

Reply:
if you are having issues with this, do change ur intellij settings to make sure it does not concatenate multiple class imports from the same package into 1